taxi_service/forms.py

```diff
@@ -0,0 +1,14 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+from django.core.validators import MinLengthValidator, MaxLengthValidator
+
+from taxi.models import Driver, Car
+
+
+class LicenseNumberValidatorMixin:
+    license_number = forms.CharField(
+        required=True,
+        validators=[MinLengthValidator(8), MaxLengthValidator(8)]
+    )

```

Comment:
The `LicenseNumberValidatorMixin` currently only checks the length of the license number. According to the task requirements, you need to validate that the license number consists of 8 characters, with the first 3 being uppercase letters and the last 5 being digits. Consider implementing a custom validator to enforce this rule.